tests/conftest.py

```diff
@@ -122,6 +123,23 @@ def ssh_client_session(sshd_addr, ssh_clientkey_path):
         del ssh_session  # noqa: WPS420
 
 
+@pytest.fixture
+def ssh_session_connect(sshd_addr, ssh_clientkey_path):
```

Comment:
Could you inject this fixture into `ssh_client_session` so there's no duplication?